src/OpenLoco/Ui/Screenshot.cpp

```diff
@@ -117,6 +91,100 @@ namespace OpenLoco::Input
             png_destroy_write_struct(&pngPtr, nullptr);
             throw;
         }
+    }
+
+    // 0x00452667
+    static std::string prepareSaveScreenshot(Gfx::RenderTarget& rt)
```

Comment:
Maybe this should just generate an output stream, and leave the writing to the caller. Thoughts?